plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java

```diff
@@ -29,7 +29,7 @@
 import org.pentaho.di.i18n.BaseMessages;
```

Comment:
Copyright date on this file needs to be updated.

Reply:
fixed